pex/bin/pex.py

```diff
@@ -302,8 +302,16 @@ def configure_clp_pex_options(parser):
       '--runtime-pex-root',
       dest='runtime_pex_root',
       default=None,
-      help='Specify the pex root to be used in the generated .pex file. [Default: ~/.pex]'
-  )
+      help='Specify the pex root to be used in the generated .pex file. [Default: ~/.pex]')
+
+  group.add_option(
+      '--strip-pex-env', '--no-strip-pex-env',
+      dest='strip_pex_env',
+      default=True,
+      action='callback',
+      callback=parse_bool,
+      help='Strip all `PEX_*` environment variables used to control the PEX runtime before handing '
+           'off control to the PEX entrypoint. [Default: %default]')
```

Comment:
It'd be helpful to put the description you wrote elsewhere here: 

> You might want to set this to `False` if this PEX executes other PEXes or the Pex CLI itself and you want the executed PEX to be controlled via PEX environment variables.

Until reading that explanation, it was not obvious to me why I would want this to be False.

Reply:
It is one of those things you don't really want to make obvious, because if you need it you know you need it - aka Pants advanced concept, but fair enough, will fix.

Reply:
Saying exactly that would help, i.e. saying it's "advanced".

> This is an advanced feature only possibly needed when using PEX to executing other PEXes or the PEX CLI itself.